product/module.go

```diff
@@ -60,6 +64,9 @@ commerce: {
 			enabled: bool | *false
 			currency: *"€" | !=""  
 		}
+		api: {
+			enabled: bool | *true
```

Comment:
Whats the point of this config flag? Flamingo commerce already has a ton of configs, in my opinion we should try to reduce them instead of adding more.

Reply:
The possibility to switch of this endpoint.
I thought that it might expose undesired data (like all possible prices)

Reply:
Okay, guess we have the same "problem" within graphql.